pkg/lang/ir/v0/cache.go

```diff
@@ -15,18 +15,11 @@
 package v0
 
 import (
-	"fmt"
-
 	"github.com/sirupsen/logrus"
 )
 
 func (g generalGraph) CacheID(filename string) string {
-	var cacheID string
-	if g.CUDA != nil {
-		cacheID = fmt.Sprintf("%s/%s-gpu", filename, g.EnvironmentName)
-	} else {
-		cacheID = fmt.Sprintf("%s/%s-cpu", filename, g.EnvironmentName)
-	}
-	logrus.Debugf("apt/pypi calculated cacheID: %s", cacheID)
```

Comment:
Does this also work for PyTorch? The CPU version and GPU version have exact the same name.

Reply:
Then have different names, such as "2.0" vs "2.0+cu113"

Reply:
SGTM.